src/main/java/org/gridsuite/caseimport/server/CaseImportController.java

```diff
@@ -36,14 +39,14 @@ public CaseImportController(CaseImportService caseImportService) {
         this.caseImportService = caseImportService;
     }
 
-    @PostMapping(value = "/cases/{caseName}", consumes = MediaType.MULTIPART_FORM_DATA_VALUE, produces = APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/cases", consumes = MediaType.MULTIPART_FORM_DATA_VALUE, produces = APPLICATION_JSON_VALUE)
     @Operation(summary = "Import a case in the parametrized directory")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The case is imported"),
         @ApiResponse(responseCode = "400", description = "Invalid case file"),
         @ApiResponse(responseCode = "422", description = "File with wrong extension"),
         @ApiResponse(responseCode = "201", description = "Case created successfully")})
     public ResponseEntity<ImportedCase> importCase(@Parameter(description = "case file") @RequestPart("caseFile") MultipartFile caseFile,
-                                                   @Parameter(description = "name of the case") @PathVariable String caseName,
+                                                   @Parameter(description = "name of the case") @NotBlank Optional<String> caseName,
```

Comment:
`NotBlank` is not a spring web annotation !
Need to do like `caseFileSource` with a `RequestParam`
Pass an optional as an argument is not a ggood pratice !